src/status_im/contexts/chat/group_create/view.cljs

```diff
@@ -0,0 +1,106 @@
+            [utils.i18n :as i18n]
+            [utils.re-frame :as rf]
+            [utils.responsiveness :as responsiveness]))
+
+(defn avatar
+  [{:keys [customization-color]}]
+  [rn/pressable {:style {:width 88 :margin-top 12 :margin-left 20}}
+   [hole-view/hole-view
+    {:holes [{:y            (- 80 32)
+              :x            (+ (- 80 32) 8)
+              :width        32
+              :height       32
+              :borderRadius 10}]}
+    [quo/group-avatar
+     {:customization-color customization-color
+      :size                :size-80}]]
+   [quo/button
+    {:on-press        (fn []
+                        #_(rf/dispatch
+                           [:show-bottom-sheet
+                            {:content (fn []
+                                        [profile-picture-picker/view
+                                         {:update-profile-pic-callback on-change-profile-pic
+                                          :has-picture?                has-picture?}])
+                             :theme   :dark
+                             :shell?  true}]))
+     :container-style {:position :absolute :right 0 :bottom 0}
+     :icon-only?      true
+     :type            :grey
+     :background      :photo
+     :size            32}
+    :i/camera]])
+
+(defn color-label
+  [theme]
+  {:color              (colors/theme-colors colors/neutral-50 colors/neutral-40 theme)
+   :padding-horizontal 20})
+
+(defn view
+  []
+  (let [theme                         (quo.theme/use-theme-value)
+        {window-width :width}         (rn/get-window)
+        profile                       (rf/sub [:profile/profile-with-image])
+        contacts                      (rf/sub [:selected-group-contacts])
+        contacts-count                (count contacts)
+        [group-name set-group-name]   (rn/use-state "")
+        group-name-empty?             (not (and (string? group-name) (not-empty group-name)))
+        [group-color set-group-color] (rn/use-state (rand-nth colors/account-colors))]
+    [floating-button-page/view
+     {:customization-color group-color
+      :gradient-cover?     true
+      :top?                true
+      :header              [quo/page-nav
+                            {:background :photo
+                             :type       :no-title
+                             :icon-name  :i/arrow-left
+                             :on-press   (fn []
+                                           (rf/dispatch [:navigate-back]))}]
+      :footer              [quo/button
+                            {:customization-color group-color
+                             :disabled?           group-name-empty?
+                             :on-press            #(debounce/throttle-and-dispatch
+                                                    [:group-chat/create group-name group-color]
+                                                    300)}
+                            (i18n/label :t/create-group-chat)]}
```

Comment:
could add use-memo to both of these to avoid re-rendering while the input changes